h/security/permissions.py

```diff
@@ -1,4 +1,16 @@
 class Permission:
+    class Group:
+        ADMIN = "group:admin"  # Is this really "EDIT" or a combination?
```

Comment:
What exactly does it let you do? I _think_ it just lets you edit the group's name and description, in which case `"group:edit"` seems appropriate. Looks like it also lets you `PATCH` to the group API though, not sure exactly what powers that gives you and whether it lets you go beyond editing group metadata like name and description

Reply:
I'm not really sure exactly. The places it's used are:

* Adds the "group_edit_url" (route "group_edit") in the activity stream data
* Allows the "group_edit" page (views/group)
* Is also used for the `GroupEditViews` in the admin pages? - I'm not sure this is anything like the other admin pages
* Also allows updating a group (views/api/group)

I think what we might have here are actually two permissions with the same name which are a bit mixed up.

I think there's a `Group.EDIT` permission which allows all the non-admin page stuff, and allows you to edit a specific group, and I think there's a separate `AdminPages.GROUPS` one as well. The ACLs granted in the group are actually irrelevant in the admin pages as the effective ACLs are defined in `Root`.

So I'm beginning to think this is actually wrong.

Reply:
Being able to edit a group on the `views/group` page seems like quite a different thing from being able to edit a group using the admin pages. That seems wrong. Feels like two different permissions should be used for those two different cases. I think that's a separate PR though, this PR didn't introduce the issue